src/rpm/signature.rs

```diff
@@ -1,7 +1,6 @@
+// re-export
 #[cfg(feature = "signing-pgp")]
```

Comment:
the feature gate should be `signing-pgp` for the whole export, but for the pgp export specifically